app/src/main/java/com/courseproject/tindar/blankNav.java

```diff
@@ -0,0 +1,65 @@
+package com.courseproject.tindar;
+
+import android.os.Bundle;
+import android.view.View;
+import android.view.Menu;
+
+import com.google.android.material.snackbar.Snackbar;
+import com.google.android.material.navigation.NavigationView;
+
+import androidx.navigation.NavController;
+import androidx.navigation.Navigation;
+import androidx.navigation.ui.AppBarConfiguration;
+import androidx.navigation.ui.NavigationUI;
+import androidx.drawerlayout.widget.DrawerLayout;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.courseproject.tindar.databinding.ActivityBlankNavBinding;
+
+public class blankNav extends AppCompatActivity {
```

Comment:
Can we change the name blackNav to BlankNavActivity everywhere? So it can be consistent in the naming with others

Reply:
updated